### PR TITLE
[DASH-2245] Fill Hosting Add/Edit dialog with useful content

### PR DESCRIPTION
**Pull request by powerczuk**
Newest version:

screen shot 2016-10-07 at 12 04 01

### CODE REVIEW

**Comment by Flixow** [comment]
Hmm, so I never think about set Domain on 'Default' based on info from sidebar and from other hand I maybe set two hostings to Default and I won't know that only one is default

File: src/apps/Hosting/HostingDialog.js [hunk 1]
```diff
@@ -94,8 +94,14 @@ const CreateHostingDialog = React.createClass({
             <Dialog.SidebarSection>
               Hosting allows you to manage, deploy and publish websites using Syncano platform.
             </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Hosting label">
+              Set the name of your hosting. Label field is not unique.
```
Comment (tomsonpl) [inline src/apps/Hosting/HostingDialog.js]: Label field doesn't have to be unique? // but do we really need this info?

File: src/apps/Hosting/HostingDialog.js [hunk 2]
```diff
@@ -94,8 +94,14 @@ const CreateHostingDialog = React.createClass({
...
+            </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Domains">
+              Define the domain names. Each name must be unique within an Instance.
```
Comment (tomsonpl) [inline src/apps/Hosting/HostingDialog.js]: 'Define the domain names' suggest that we have many names for one domain. 
What do you think about just "Define the domains"? not too confusing?

Comment (powerczuk) [inline src/apps/Hosting/HostingDialog.js]: 👍

**Comment by tomsonpl** [comment]
@Flixow I am not sure if I understood you correctly, but if you want to set a domain of another hosting to default, you will get an error saying more or less this " default already in use"

**Comment by powerczuk** [comment]
screen shot 2016-10-05 at 17 09 22

**Comment by wpasinski** [comment]
how about using italic font for key words?

zrzut ekranu 2016-10-05 o 19 50 36

**Comment by powerczuk** [comment]
Good idea @wpasinski, I'll use italic font instead of apostrophe

File: src/apps/Hosting/HostingDialog.js [hunk 3]
```diff
@@ -94,8 +97,16 @@ const CreateHostingDialog = React.createClass({
...
+              You can set the name of your hosting.
+            </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Domains">
+              {`Define the domains. Each name must be unique within an Instance. You can also
+              name your domain as 'default' then it will be connected directly to your current Instance
+              and avaliable at https://${currentInstanceName}.syncano.site`}
+            </Dialog.SidebarSection>
             <Dialog.SidebarSection last={true}>
-              <Dialog.SidebarLink to="http://docs.syncano.io/v1.1/docs/">
+              <Dialog.SidebarLink to="http://docs.syncano.io/v1.1/docs/hosting">
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingDialog.js]: `/` missing at the end of the link

File: src/apps/Hosting/HostingDialog.js [hunk 4]
```diff
@@ -94,8 +97,16 @@ const CreateHostingDialog = React.createClass({
             <Dialog.SidebarSection>
               Hosting allows you to manage, deploy and publish websites using Syncano platform.
             </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Hosting label">
+              You can set the name of your hosting.
+            </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Domains">
+              {`Define the domains. Each name must be unique within an Instance. You can also
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingDialog.js]: i think we should also mention about the structure of the domain, `https://${currentInstanceName}--{domain}.syncano.site`

File: src/apps/Hosting/HostingDialog.js [hunk 5]
```diff
@@ -94,8 +97,16 @@ const CreateHostingDialog = React.createClass({
             <Dialog.SidebarSection>
               Hosting allows you to manage, deploy and publish websites using Syncano platform.
             </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Hosting label">
+              You can set the name of your hosting.
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingDialog.js]: but why user would like to set the name of his hosting?

**Comment by powerczuk** [comment]
screen shot 2016-10-06 at 13 27 52

**Comment by powerczuk** [comment]
I made another update of description, because Tomek is working on check button for setting up  a `default domain`

screen shot 2016-10-06 at 14 39 46

File: src/apps/Hosting/HostingDialog.js [hunk 6]
```diff
@@ -92,10 +96,21 @@ const CreateHostingDialog = React.createClass({
         sidebar={
           <Dialog.SidebarBox>
             <Dialog.SidebarSection>
-              Hosting allows you to manage, deploy and publish websites using Syncano platform.
+              Hosting allows you to manage, deploy and publish websites using Syncano Platform.
+            </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Hosting label">
+              Name of the hosthing in Syncano Dashboard.
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingDialog.js]: `hosthing`

File: src/apps/Hosting/HostingDialog.js [hunk 7]
```diff
@@ -92,10 +96,21 @@ const CreateHostingDialog = React.createClass({
...
+            </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Domains">
+              Define the domains which will be linked to your hosting at
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingDialog.js]: and what is the motivation/use case to define domains?

**Pull request merged**